src/plone/base/interfaces/controlpanel.py

```diff
@@ -399,8 +399,10 @@ class ITinyMCELayoutSchema(Interface):
             "This option enables you to specify CSS files "
             "that will be used within the the editable area of the editor "
             "(e.g. ++plone++mystyles/tinymce.css). "
+            "These styles will also appear in the Format list of TinyMCE."
             "In addition to what is listed here, "
-            "the barceloneta bundle CSS is also added."
+            "the CSS file defined under 'tinymce-content-css'"
```

Comment:
Wew lads I did not know this was possible.  Imma change my policy products to use this!

Reply:
Turns out this made me find a bug haha.

https://github.com/plone/Products.CMFPlone/pull/3638

What a wild goose chase!